lib/ueberauth/strategy/github.ex

```diff
@@ -158,7 +158,7 @@ defmodule Ueberauth.Strategy.Github do
     %Info{
       name: user["name"],
       nickname: user["login"],
-      email: user["email"],
+      email: user["email"] || Enum.find(user["emails"] || [], &(&1["primary"]))["email"],
```

Comment:
`user["emails"]` is an array of map objects where one of them is marked with `%{primary: true}`. Yes I suspect that it could return nil but `nil["foo"]` will just return `nil` as well 😄